<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>756</width>
    <height>738</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DYJ的sina搜索</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="letKeyword">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>411</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>（请在此处输入关键字）</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSearch">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>40</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>开始搜索</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblSearch">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>关键字：</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="tbrList">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>491</width>
      <height>611</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>756</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>letKeyword</sender>
   <signal>returnPressed()</signal>
   <receiver>btnSearch</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>390</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>508</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>search()</slot>
 </slots>
</ui>
